netlify.toml

```diff
@@ -1,16 +1,128 @@
+[functions]
```

Comment:
this shouldn't be committed - this is an unfortunately side effect of using the Netlify CLI and accidentally having it open and saving the results, or it sometimes occur if it fails during that process as what netlify does is overwrite the file with more configuration during their build/deploy process

can you revert this?

Reply:
Totally!